### ISSUE TITLE
Cannot use library at all every week

### ISSUE THREAD

**Issue by raiyni** [derek-adair/nflgame#101]
I can't use nflgame until I comment out the line that is erroring in this stack trace. After I comment it out and run once, then I can re-add it back.

```$ python nflgame-update-schedule --rebuild
Traceback (most recent call last):
  File "nflgame-update-schedule", line 3, in <module>
    import nflgame.update_sched
  File "C:\python3\lib\site-packages\nflgame\__init__.py", line 95, in <module>
    import nflgame.game  # noqa
  File "C:\python3\lib\site-packages\nflgame\game.py", line 15, in <module>
    import nflgame.sched
  File "C:\python3\lib\site-packages\nflgame\sched.py", line 101, in <module>
    games, last_updated = _create_schedule()
  File "C:\python3\lib\site-packages\nflgame\sched.py", line 81, in _create_schedule
    missing_weeks = check_missing_weeks(sched, year, phase)
  File "C:\python3\lib\site-packages\nflgame\sched.py", line 38, in check_missing_weeks
    missing_weeks.remove(stored_week)
ValueError: list.remove(x): x not in list
```

**Comment by derek-adair** [comment]
Suspected duplicate of #97.  I'm actually unable to even reach this error message tho...

Which nflgame-redux version are you using?

**Comment by derek-adair** [comment]
Alrighty i found this when testing out a fresh install.  It appears that the update team logic is broken.  The URI we used to scrape is redirecting to a different URL that does not contain the rosters anylonger.

Stay tuned I'll swap out this logic to access the feeds-rs right now.

**Comment by derek-adair** [comment]
Released under 2.0.3.
